runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -541,4 +642,89 @@ private static HiscoreEndpoint toEndPoint(final AccountType accountType)
 		private final String name;
 		private final HiscoreEndpoint endpoint;
 	}
+
+	private static String longBossName(String boss)
+	{
+		switch (boss.toLowerCase())
+		{
+			case "corp":
+				return "Corporeal Beast";
+
+			case "jad":
+				return "TzTok-Jad";
+
+			case "kq":
+				return "Kalphite Queen";
+
+			case "chaos ele":
+				return "Chaos Elemental";
+
+			case "dusk":
+			case "dawn":
+			case "galio":
```

Comment:
even though this is funny, I dont think we should use this. Galio is reference to League of Legends character that is big gargoyle